server/index.js

```diff
@@ -1,18 +1,20 @@
-const express = require('express')
-// var bodyParser = require('body-parser')
-const app = express()
+var express = require('express')
+var app = express()
 // app.use(bodyParser.json())
-const port = 3000;
-const mongoose = require('mongoose');
-// var faker = require('faker');
-// mongoose.connect('mongodb://localhost/test', {useNewUrlParser: true});
-// const DB = require('./database');
+var port = 3007;
+
+var DB = require('/Users/sinamb/Documents/Programs/SinaService/database/index.js');
```

Comment:
The file path needs to be relative to the current directory - if someone else was to fork your project and try to run it it won't work. Like so - `../database/index.js`